catalog/urls.py

```diff
@@ -0,0 +1,12 @@
+from urllib.request import url2pathname
+from django.urls import path
+
+from catalog import views
+
+urlpatterns = [
+    path('satellite',
+         views.SatelliteView.as_view(),
+         name='Satellite-catalog'
```

Comment:
Change "Satellite" from "satellite" to keep the lowercase pattern of the other tabs

Reply:
Adjusted.